.github/ISSUE_TEMPLATE/undergrad-research.md

```diff
@@ -32,9 +36,10 @@ We are thrilled that you're interested in research in our research group! Please
 (It's OK to say, "I don't know; I'm looking to explore!")
 
 
-## Optional Stuff
+## Background
 
-(Feel free to leave any of these blank.)
+- **Freshmen or Sophomore**: The following are optional.
+- **Junior or above**: The first two questions are required, others are optional.
```

Comment:
I think this is a good idea. We can be forgiving with the answers but getting people to at least try to find an interesting paper would be useful.

Here's an alternative wording that has the bonus of avoiding the weirdly gendered name for first-year college students. :smiley:

> If you're in your first or second year at Cornell, these are all optional. If you're a junior or senior, please answer the first two (the rest is still optional).

Reply:
Addressed